beetsplug/lyrics.py

```diff
@@ -441,7 +446,103 @@ def _scrape_lyrics_from_html(self, html):
                 footer.replace_with("")
         return lyrics_div.get_text()
 
+class Tidal(Backend):
+    REQUIRES_BS = False
+    
+    def __init__(self, config, log):
+        super().__init__(config, log)
+        self._log = log
+        
+        sessionfile = self.config["tidal_session_file"].get(
+            confuse.Filename(in_app_dir=True)
+        )
 
+        self.session = self.load_session(sessionfile)
+       
+        self.attempts = self.config["tidal_attempts"].get(int)
```

Comment:
Please add default values so that these fields can be made optional in the config.